### PR TITLE
Add BROTHERHOBBYF405V3

### PR DESCRIPTION
**Pull request by H-ChuWu**
Add new target BROTHERHOBBYF405V3

### CODE REVIEW

File: configs/BROTHERHOBBYF405V3/config.h [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F405
+#define BOARD_NAME        BrotherHobbyF405V3
```
Comment (haslinghuis) [inline configs/BROTHERHOBBYF405V3/config.h:25]:
```suggestion
#define BOARD_NAME        BROTHERHOBBYF405V3
```

File: configs/BROTHERHOBBYF405V3/config.h [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+#define SPI1_SDI_PIN            PA6
+#define SPI2_SDI_PIN            PB14
+#define SPI3_SDI_PIN            PB4
+#define SPI1_SDO_PIN            PA7
+#define SPI2_SDO_PIN            PB15
+#define SPI3_SDO_PIN            PB5
+#define ADC_VBAT_PIN            PC0
+#define ADC_RSSI_PIN            PC5
+#define ADC_CURR_PIN            PC1
+#define PINIO1_PIN 				PC14
```
Comment (haslinghuis) [inline configs/BROTHERHOBBYF405V3/config.h:76]:
```suggestion
#define PINIO1_PIN              PC14
```

File: configs/BROTHERHOBBYF405V3/config.h [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+#define DEFAULT_CURRENT_METER_SOURCE        CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE        VOLTAGE_METER_ADC
+#define DEFAULT_CURRENT_METER_SCALE         386
+#define BEEPER_INVERTED
+#define FLASH_SPI_INSTANCE                  SPI3
+#define MAX7456_SPI_INSTANCE                SPI2   
+#define GYRO_1_SPI_INSTANCE                 SPI1
+#define GYRO_1_ALIGN                        CW90_DEG
+#define SYSTEM_HSE_MHZ                      8  
+#define DASHBOARD_I2C_INSTANCE 				(I2CDEV_1)
```
Comment (haslinghuis) [inline configs/BROTHERHOBBYF405V3/config.h:109]: Can be removed
```suggestion
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]
Quick review - did not check dma / timers

**Comment by H-ChuWu** [comment]
> 快速回顾 - 没有检查 dma / 计时器

File: configs/BROTHERHOBBYF405V3/config.h [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+#define ADC_RSSI_PIN            PC5
+#define ADC_CURR_PIN            PC1
+#define PINIO1_PIN              PC14
+#define FLASH_CS_PIN            PA15
+#define MAX7456_SPI_CS_PIN      PB12
+#define GYRO_1_EXTI_PIN         PC4
+#define GYRO_1_CS_PIN           PA4
+
+#define TIMER_PIN_MAPPING \
+ 	TIMER_PIN_MAP(0, PA8, 1, 0) \
```
Comment (ot0tot) [inline configs/BROTHERHOBBYF405V3/config.h:83]: Change to `TIMER_PIN_MAP(0, PA8, 1, 1) \`

File: configs/BROTHERHOBBYF405V3/config.h [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+ 	TIMER_PIN_MAP(1, PA9, 1, 1) \
+ 	TIMER_PIN_MAP(2, PB10, 1, 0) \
+ 	TIMER_PIN_MAP(3, PB11, 1, 0) \
```
Comment (ot0tot) [inline configs/BROTHERHOBBYF405V3/config.h:86]: Change to `TIMER_PIN_MAP(3, PB11, 1, 1) \`

File: configs/BROTHERHOBBYF405V3/config.h [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+ 	TIMER_PIN_MAP(4, PB7, 1, 0) \
+ 	TIMER_PIN_MAP(5, PB6, 1, 0) \
+ 	TIMER_PIN_MAP(6, PB0, 2, 0) \
+ 	TIMER_PIN_MAP(7, PB1, 2, 0) \
+ 	TIMER_PIN_MAP(8, PC9, 2, 0)
+
+#define SPI2_TX_DMA_OPT     0
```
Comment (ot0tot) [inline configs/BROTHERHOBBYF405V3/config.h:93]: Delete `#define SPI2_TX_DMA_OPT     0`

**Review by ot0tot — CHANGES_REQUESTED** [review]

File: configs/BROTHERHOBBYF405V3/config.h [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F405
+#define BOARD_NAME        BHF405V3
```
Comment (haslinghuis) [inline configs/BROTHERHOBBYF405V3/config.h:25]: Does not match 

image

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Review by ot0tot — APPROVED** [review]

**Pull request merged**